Sources/Model/GraphQL/HyperwalletTransferMethodConfigurationField.swift

```diff
@@ -42,7 +42,7 @@ final class TransferMethodConfigurationFieldResult: HyperwalletTransferMethodCon
     /// - Parameters:
     ///   - transferMethodUIConfigurations: the GraphQL `[TransferMethodUIConfiguration]`
```

Comment:
update HyperwalletTransferMethodConfiguration in all the comments as well

Reply:
Updated.